### DISCUSSION TITLE
need help!!! Truffle test have some weird bug

### DISCUSSION THREAD

**Discussion by DayLight109** [ConsenSys-archive/truffle#6209] [Ideas]
i have some project to fix 
 Contract: New OverFlow
     Error: NewTimeLock has not been deployed to detected network (network/artifact mismatch)
i checked 
1.Truffle-Config
2.test js
3.Contract
it still said  Contract: New OverFlow
     Error: NewTimeLock has not been deployed to detected network (network/artifact mismatch)
need help!!!

**Comment by Oswal22** [comment]
How can I identify it. 
I can realized that 2 networks ware available but one of them was hidden always, I join the hidden network ( wrong of my)later , go back to connect the usual but then someone change the name of that network before I hidden it again ,so only 2 devices could connected but I can't visualize it any more ; so the only  device connected try to stole or limited me the access but that network keep off line, today tried to do the switch but in my PC I can't use the command to forget past networks. Hope you mean that
